packages/framework/ai-collab/src/explicit-strategy/debugEvents.ts

```diff
@@ -0,0 +1,253 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { v4 as uuidv4 } from "uuid";
+
+import type { DebugEvent, EventFlowDebugEvent } from "../aiCollabApi.js";
+
+import type { TreeEdit } from "./agentEditTypes.js";
+
+/**
+ * A debug event for signaling the start of the ai-collab's core event loop.
+ * Which makes various calls to the LLM to eventually apply edits to the users SharedTree which
+ * accomplish the user's provided goal.
+ * @alpha
+ */
+export interface CoreEventLoopStartedDebugEvent extends EventFlowDebugEvent {
```

Comment:
Nit: what do we think of removing `DebugEvent` from all the child interface names?